packages/react-ui/src/components/molecules/AccordionGroup.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Accordion, AccordionDetails, AccordionSummary } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+const AccordionContainer = styled('div', {
+  shouldForwardProp: (prop) => prop !== 'variant'
+})(({ variant, theme }) => ({
+  width: '100%',
+  borderRadius: theme.spacing(0.5),
+
+  ...(variant === 'outlined' && {
+    backgroundColor: theme.palette.background.paper,
+    boxShadow: `inset 0 0 0 1px ${theme.palette.divider}`
```

Comment:
Out of curiosity.

We've got `theme.shadows`. Are they of no use here? If not, when they should be used (they are used in extensively in cloud-native)